portfolio.css

```diff
@@ -0,0 +1,305 @@
+*{
+	box-sizing: border-box;
+}
+
+/* this is for navbar (every file should have this) */
+body {
+	margin: 0;
+	background-color: #F0F8FF;
+}
+
+nav {
+	display: flex;
+	justify-content: space-between;
+	background-color: #006699;
+}
+
+.name {
+	display: flex;
+	align-items: center;
+	margin: 10px;
+	color: #FFE4E1;
+	font-size: 1.25em;
+	padding: 0px;
+	height: 40px;
+}
+
+.navlinks{
+	display: flex;
+	align-items: center;
+	justify-content: flex-end;
+}
+
+ul{
+	list-style-type: none;
+	justify-content: flex-end;
+	padding: 0px;
+}
+
+.navlinks li {
+		display: flex;
+		display: inline;
+		align-items: center;
+		padding: 0px 20px 0px 0px;
+}
+
+.navlinks a {
+	text-decoration: none;
+	border-bottom: 1px solid #FFE4E1;
+	color: #FFE4E1;
+}
+
+.navlinks a:hover {
+	border-bottom: 3px solid #FFE4E1;
+}
+
+
+
+/*portfolio pages content */
+
+
+h1 {
+	text-align: center;
+	margin: 0px;
+	padding: 20px;
+}
+
+h2 {
+	text-decoration: underline;
+	text-align:center;
+}
+
+.content img {
+	display:flex;
+	width: 300px;
+	height: 200px;
+}
+
+.example {
+	display: flex;
+	justify-content: center;
+	margin: 0px;
+	flex-wrap: wrap;
+}
+
+.written_content {
+	display:flex;
+}
+
+p {
+	margin: 0px 50px 20px 10px;
+	width: 200px;
+}
+
+.example  ul {
+	margin: 0px 50px 20px 0px;
+}
+
+.content {
+	color: #006699;
+}
+
+h3 {
+	text-decoration: underline;
+	text-align:center;
+}
+
+.certificates {
+	display:inline-grid;
```

Comment:
I think just using display: grid should do the trick for this section, I don't think you need inline grid. 

Inline grid is usually used if you need to have two grids sat next to each other (weird)